src/emqtt_sock.erl

```diff
@@ -160,3 +161,9 @@ apply_host_check_fun(Opts) ->
                               public_key:pkix_verify_hostname_match_fun(https)}]},
             [DefHostCheck | Opts]
     end.
+
+maybe_drop_incompatible_options(['tlsv1.3'], SslOpts) ->
+    Incompatible = [reuse_sessions, secure_renegotiate],
```

Comment:
just curious to know why they are Incompatible?

Reply:
https://www.erlang.org/doc/apps/ssl/using_ssl.html#session-tickets-and-session-resumption-in-tls-1.3

Reply:
https://www.erlang.org/doc/man/ssl.html#renegotiate-1